### PR TITLE
BIP171 - Adding Cerise to Reference implementation

### PR DESCRIPTION
**Pull request by straumat**

### CODE REVIEW

**Comment by straumat** [comment]
Up ?

File: bip-0171.mediawiki [hunk 1]
```diff
@@ -193,7 +193,15 @@ While this new standard is adopted, software and providers can continue to use a
 
 ==Reference implementation==
 
-TODO
+===Cerise===
```
Comment (luke-jr) [inline bip-0171.mediawiki]: In English please

Comment (luke-jr) [reply bip-0171.mediawiki]: I don't know what "cerise" is, and a dictionary doesn't help...

**Comment by straumat** [comment]
Cerise is then ame of the implementation I made :
https://github.com/straumat/cerise
Anyway, i think you can cancel my merge request. Time flies and my
contribution is certainly no more usefull

**Comment by luke-jr** [comment]
Time may fly, but not for this BIP. Nothing has changed much since previously.

You can close it if you want.

**Pull request closed without merging**